### PR TITLE
Code to check for armstrong numbers

### PR DESCRIPTION
**Pull request by Akarsh-dundun**

### CODE REVIEW

File: week1/armstrong.rb [hunk 1]
```diff
@@ -9,5 +9,16 @@
...
+    sum = 0
+    for i in l do
+        sum += i**3
+    end
+    
+    if sum == n
+        return true
+        
+    else
+        return false
```
Comment (raosush) [inline week1/armstrong.rb]:
```suggestion
    if sum == n
        return true
    else
        return false
    end
```
Syntax error -> `if` block should have an ending `end`

File: week1/armstrong.rb [hunk 2]
```diff
@@ -9,5 +9,25 @@
...
+    
+    if n < 0
+        return false
+        
+    else
+
+        l = n.digits
+        sum = 0
+        for i in l do
+            sum += i**3
```
Comment (gaurang2001) [inline week1/armstrong.rb]: the problem statement asks for sum of digits raised to number of digits in the number. So this `3` should be replaced by `l.length` (`sum += i**l.length`). make this one change and push again

**Review by gaurang2001 — CHANGES_REQUESTED** [review]

**Review by gaurang2001 — APPROVED** [review]

**Comment by gaurang2001** [comment]
You've completed the assignment. Good Job!! 🎉

**Comment by Akarsh-dundun** [comment]
Good Evening Gaurang,    Just writing to apologize for the really ignorant submission of this weeks activity. I will try to tone down on the errors going forward RegardsAkarsh Gupta Sent from Mail for Windows From: Gaurang VelingkarSent: 04 February 2022 21:20To: IRIS-NITK/IRIS-RoR-Bootcamp-2021Cc: Akarsh Gupta; AuthorSubject: Re: [IRIS-NITK/IRIS-RoR-Bootcamp-2021] Code to check for armstrong numbers (PR #3) Closed #3.—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>

**Pull request closed without merging**